doc/040_backup.rst

```diff
@@ -437,6 +437,12 @@ written, and the next backup needs to write new metadata again. If you really
 want to save the access time for files and directories, you can pass the
 ``--with-atime`` option to the ``backup`` command.
 
+Note that ``restic`` does not back up all metadata associated with files. Of
+particular note are::
+
+  - inode flags on Unix platforms
+  - the "hidden" flag on Windows (see #2075)
```

Comment:
A link to an issue has to look like this;
```
`#2075<https://github.com/restic/restic/issues/2075>`_
```

As far as I remember, on windows also no ACLs or file ownership are backed up.